matter/wifi/rs911x/rsi_if.c

```diff
@@ -45,76 +45,80 @@ bool hasNotifiedWifiConnectivity = false;
+    extra_info->beacon_lost_count =
+        test->beacon_lost_count - temp_reset->beacon_lost_count;
+    extra_info->beacon_rx_count =
+        test->beacon_rx_count - temp_reset->beacon_rx_count;
+    extra_info->mcast_rx_count =
+        test->mcast_rx_count - temp_reset->mcast_rx_count;
+    extra_info->mcast_tx_count =
+        test->mcast_tx_count - temp_reset->mcast_tx_count;
+    extra_info->ucast_rx_count =
+        test->ucast_rx_count - temp_reset->ucast_rx_count;
+    extra_info->ucast_tx_count =
+        test->ucast_tx_count - temp_reset->ucast_tx_count;
+    extra_info->overrun_count = test->overrun_count - temp_reset->overrun_count;
   }
   return status;
 }
-int32_t wfx_rsi_reset_count()
-{
+int32_t wfx_rsi_reset_count() {
   int32_t status;
-  uint8_t buff[28] = { 0 };
-  status           = rsi_wlan_get(RSI_WLAN_EXT_STATS, buff, sizeof(buff));
+  uint8_t buff[BUFF_SIZE_28] = {0};
+  status = rsi_wlan_get(RSI_WLAN_EXT_STATS, buff, sizeof(buff));
   if (status != RSI_SUCCESS) {
     WFX_RSI_LOG("\r\n Failed, Error Code : 0x%lX\r\n", status);
   } else {
-    rsi_wlan_ext_stats_t *test    = (rsi_wlan_ext_stats_t *)buff;
+    rsi_wlan_ext_stats_t *test = (rsi_wlan_ext_stats_t *)buff;
     temp_reset->beacon_lost_count = test->beacon_lost_count;
-    temp_reset->beacon_rx_count   = test->beacon_rx_count;
-    temp_reset->mcast_rx_count    = test->mcast_rx_count;
-    temp_reset->mcast_tx_count    = test->mcast_tx_count;
-    temp_reset->ucast_rx_count    = test->ucast_rx_count;
-    temp_reset->ucast_tx_count    = test->ucast_tx_count;
-    temp_reset->overrun_count     = test->overrun_count;
+    temp_reset->beacon_rx_count = test->beacon_rx_count;
+    temp_reset->mcast_rx_count = test->mcast_rx_count;
+    temp_reset->mcast_tx_count = test->mcast_tx_count;
+    temp_reset->ucast_rx_count = test->ucast_rx_count;
+    temp_reset->ucast_tx_count = test->ucast_tx_count;
+    temp_reset->overrun_count = test->overrun_count;
   }
   return status;
 }
-int32_t wfx_rsi_disconnect(){
-	int32_t status = rsi_wlan_disconnect();
-	return status;
+int32_t wfx_rsi_disconnect() {
+  int32_t status = rsi_wlan_disconnect();
+  return status;
 }
-static void wfx_rsi_join_cb(uint16_t status, const uint8_t *buf, const uint16_t len)
-{
+static void wfx_rsi_join_cb(uint16_t status, const uint8_t *buf,
+                            const uint16_t len) {
   WFX_RSI_LOG("%s: status: %02x", __func__, status);
   wfx_rsi.dev_state &= ~WFX_RSI_ST_STA_CONNECTING;
   temp_reset = (wfx_wifi_scan_ext_t *)malloc(sizeof(wfx_wifi_scan_ext_t));
-  memset(temp_reset,0,sizeof(wfx_wifi_scan_ext_t));
+  memset(temp_reset, CLEAR_BUFFER, sizeof(wfx_wifi_scan_ext_t));
   if (status != RSI_SUCCESS) {
     /*
      * We should enable retry.. (Need config variable for this)
      */
     WFX_RSI_LOG("%s: failed. retry: %d", __func__, wfx_rsi.join_retries);
-#if (WFX_RSI_CONFIG_MAX_JOIN != 0)
+#if (WFX_RSI_CONFIG_MAX_JOIN != JOIN_CNT0)
```

Comment:
Like previously mentioned in those case, `0` doesn't need to be a define.

Reply:
kept 0 only